src/views/product-tutorials-view/components/sitemap/index.tsx

```diff
@@ -9,20 +9,30 @@ import {
   ProductOption,
   TutorialLite as ClientTutorialLite,
 } from 'lib/learn-client/types'
-import { ProductPageData } from 'views/product-tutorials-view/server'
 import Heading from 'components/heading'
 import IconTile from 'components/icon-tile'
 import { IconCollections24 } from '@hashicorp/flight-icons/svg-react/collections-24'
 import { getSitemapHeading } from 'views/product-tutorials-view/helpers/heading-helpers'
 import s from './sitemap.module.css'
 
+export interface ProductTutorialsSitemapProps {
```

Comment:
⛏️ Nit, non-blocking: we could move this to a separate `types.ts` file if we want, but personally totally fine with leaving it inline 👍